src/components/App.tsx

```diff
@@ -5404,6 +5405,23 @@ class App extends React.Component<AppProps, AppState> {
     if (probablySupportsClipboardWriteText && elements.length > 0) {
       options.push(actionCopyAsSvg);
     }
+
+    const selectedElements = getSelectedElements(elements, this.state);
+
+    let allowBinding = false;
+    if (selectedElements.length === 2) {
+      const textElement =
+        isTextElement(selectedElements[0]) ||
+        isTextElement(selectedElements[1]);
+
+      const bindingContainer =
+        isTextBindableContainer(selectedElements[0]) ||
+        isTextBindableContainer(selectedElements[1]);
+      if (textElement && bindingContainer) {
+        allowBinding = true;
+      }
+    }
```

Comment:
1) let's put this into `action.contextMenuPredicate` instead
2) as discussed on chat, if we're not gonna allow replacing for now, then we should not show the item if the container already has a bound text